tools/link.py

```diff
@@ -1608,7 +1608,7 @@ def limit_incoming_module_api():
       # do anything (as the user's list won't contain these functions), and if
       # we did add them, the pass would assert on incompatible lists, hence the
       # condition in the above if.
-      settings.ASYNCIFY_REMOVE += emscripten.ASAN_C_HELPERS
+      settings.ASYNCIFY_REMOVE.append("__asan_*")
```

Comment:
Is this change needed?

Reply:
Yeah, ASan's C API calls into other C APIs and then I was getting test failures from Asyncify trying to transform more of ASan internals.

It's easier and more reliable to cover all of ASan APIs at once.

Reply:
sgtm